### PR TITLE
Blog - Customizing the Django Admin

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by balapriyac** [comment]
Hi @msali123 can you please add the preview link here? Thank you.

**Comment by msali123** [comment]
@balapriyac I don't know why but it didn't generate any link to preview.

**Comment by balapriyac** [comment]
Yeah, I noticed that, too. I wasn't able to find it in the build summary
either.

Is it possible to push another commit with a minor edit & see if it
generates preview for the build?

**Comment by msali123** [comment]
Preview URL:
https://6405aa15ac95be58b7619d52--ephemeral-narwhal-3ac3b2.netlify.app

@balapriyac yup that works. I've updated it from main branch.

File: blog/_posts/2023-04-13-customize-django-admin-site.md [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+
+Objects can refer to other objects using foreign keys. The post model has a many-to-one relationship with the author via ForeignKey which is displayed as an unclickable text on the admin site. The title fields of the Post model are clickable, making it easy to change their details. If you want to change the author's details, you must go to the Author admin page to [make](/blog/using-cmake) the relevant changes. This can become tedious if a lot of changes are to be made.
+
+<div class="wide">
+![unclickable]({{site.images}}{{page.slug}}/iKdP0kL.png)\
+</div>
+
+However, you can use Django's [URL-reversing](https://docs.djangoproject.com/en/4.1/ref/urlresolvers/) system to access the related object's admin page and [make](/blog/using-cmake) the changes more efficiently. URL reversing refers to the process of converting a named URL pattern to a URL string that can be used in an HTTP request. This makes it easier to link between pages, as you can refer to named URL patterns instead of hardcoded URLs.
+
+~~~{.python caption="mosels.py"}
```
Comment (balapriyac) [inline blog/_posts/2023-04-13-customize-django-admin-site.md:387]:
```suggestion
~~~{.python caption="models.py"}
```

**Review by balapriyac — CHANGES_REQUESTED** [review]

File: blog/_posts/2023-04-13-customize-django-admin-site.md (hunk 1 above)
Comment (msali123) [reply blog/_posts/2023-04-13-customize-django-admin-site.md:387]: @balapriyac just this issue?

**Comment by balapriyac** [comment]
Yes, Sameer. Just this typo. Everything else looks good.  Great work on
this!

**Comment by balapriyac** [comment]
@jalletto this is ready to merge!

**Review by jalletto — APPROVED** [review]

**Pull request merged**